python/ray/dashboard/client/src/api.ts

```diff
@@ -283,3 +283,46 @@ export const setTuneExperiment = (experiment: string) =>
 
 export const enableTuneTensorBoard = () =>
   post<{}>("/api/enable_tune_tensorboard", {});
+
+export type MemoryTableSummary = {
+  total_actor_handles: number;
+  total_captured_in_objects: number;
+  total_local_ref_count: number;
+  total_object_size: number;
```

Comment:
It would be nice if this name contained the unit of measurement, but a comment instead would do alright (I don't think it's worth another backend change to make them match).